README.md

```diff
@@ -1,8 +1,362 @@
+constexpr int block_size = 256;
+
+__global__ void reduce(int const* data, int* result, std::size_t N) {
+  using BlockReduce = cub::BlockReduce<int, block_size>;
+  __shared__ typename BlockReduce::TempStorage temp_storage;
+
+  int index = threadIdx.x + blockIdx.x * blockDim.x;
+  int sum = 0;
+  if (index < N) {
+    sum += data[index];
+  }
+  sum = BlockReduce(temp_storage).Sum(sum);
+
+  if (threadIdx.x == 0) {
+    cuda::atomic_ref<int, cuda::thread_scope_device> atomic_result(*result);
+    atomic_result.fetch_add(sum, cuda::memory_order_relaxed);
+  }
+}
+
+int main() {
+  std::size_t N = 1000;
+  // Allocate and initialize input data
+  thrust::device_vector<int> data(N);
+  thrust::fill(data.begin(), data.end(), 1);
+  // Allocate output data
+  thrust::device_vector<int> kernel_result(1);
+
+  // Compute the sum reduction of `data` using a custom kernel
+  int num_blocks = (N + block_size - 1) / block_size;
+  reduce<<<num_blocks, block_size>>>(thrust::raw_pointer_cast(data.data()),
+                                     thrust::raw_pointer_cast(kernel_result.data()),
+                                     N);
+
+  auto err = cudaDeviceSynchronize();
+  if (err != cudaSuccess) {
+    std::cout << "Error: " << cudaGetErrorString(err) << std::endl;
+    return -1;
+  }
+
+  // Compute the same sum reduction using Thrust
+  int thrust_result = thrust::reduce(thrust::device, data.begin(), data.end(), 0);
+
+  // Ensure the two solutions are identical
+  std::printf("Custom Kernel Sum: %d\n", kernel_result[0]);
+  std::printf("Thrust reduce Sum: %d\n", thrust_result);
+  assert(kernel_result[0] == thrust_result);
+  return 0;
+}
+```
+
+## Getting Started
+
+### Users
+
+Generally speaking, because CCCL is a header-only library, users need only concern themselves with how they get the header files and how they incorporate them into their build system. 
+Anyone interested in using CCCL in their CUDA C++ application can get started by referring to the information below. 
+
+#### CUDA Toolkit 
+The easiest way to get started using CCCL is if you already have the [CUDA Toolkit](https://developer.nvidia.com/cuda-toolkit), then you already have the CCCL headers installed on your system.
+When you compile with `nvcc` you can simply include the desired headers in your code with no additional configuration required.
+If compiling with another compiler, you will need to add the include path (e.g., `/usr/local/cuda/include`) to the CCCL headers to your build system.
+
+```cpp
+#include <thrust/device_vector.h>
+#include <cub/cub.cuh>
+#include <cuda/std/atomic>
+```
+
+#### GitHub
+
+For users that want to stay on the cutting edge of CCCL development, we actively support and encourage users to use CCCL from GitHub. 
+We support using a newer version of CCCL with an older version of the CUDA Toolkit, but not the other way around.
+For complete information on compatibility between CCCL and the CUDA Toolkit, see [our platform support](#platform-support).
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**💡 suggestion:** ‏ 
```suggestion
Users that want to stay on the cutting edge of CCCL development, are supported and encouraged to use CCCL from GitHub. 
Using a newer version of CCCL with an older version of the CUDA Toolkit is supported, but not the other way around.
For complete information on compatibility between CCCL and the CUDA Toolkit, see [platform support](#platform-support).
```

Reply:
I like keeping both to show the welcoming attitude.

Reply:
I like shortening to encouraged since "supported" is in the following sentence.